src/q7/radiation-ELEC2795/exam/2019/Janvier/All/radiation-ELEC2795-exam-2019-Janvier-All.tex

```diff
@@ -71,7 +162,29 @@ \section{(4,5 points)}
     \end{itemize}
 \end{enumerate}
 
-\nosolution
+\begin{solution}
+
+\begin{enumerate}
+ \item 
+ \begin{align*}
+  \hat{I}_m &= \frac{\bm{h}^H \bm{y}}{\en_u \norm{\bm{h}}^2}\\
+  P_N &= \frac{2 N_0}{\en_u \norm{\bm{h}}^2}\\
+  P_S &= \sigma_I^2\\
+  \mathrm{SNR} &= \frac{P_S}{P_N} = \frac{\en_s \norm{\bm{h}}^2}{N_0}
+ \end{align*}
+ The SNR is increased by the diversity (use of multiple receivers).
+ \item 
+ \begin{align*}
+  \hat{I}_m &= \frac{\sum_{i=1}^2 h_i^* y_i / \sigma_{w_i'}^2}{\en_u \sum_{i=1}^2 \abs{h_i}^2 / \sigma_{w_i'}^2}\\
+  P_N &= \frac{1}{\en_u^2 \sum_{i=1}^2 \abs{h_i}^2 / \sigma_{w_i'}^2} = \frac{1}{\en_u \sum_{i=1}^2 \abs{h_i}^2 / (2 N_i)}\\
+  P_S &= \sigma_I^2\\
+  \mathrm{SNR} &= \frac{P_S}{P_N} = \en_s \sum_{i=1}^2 \frac{\abs{h_i}^2}{N_i}
+ \end{align*}
+ The SNR increases when multiple receivers are used. If one antenna has a very large noise ($N_1 \gg N_2$), its associated term in the SNR drops to zero. This antenna is thus not used for the symbol prediction.
+\end{enumerate}
+
+\end{solution}
+
 
 \section{(4,5 points)}
```

Comment:
There is a MCQ environment for this kind of things (even though it is pretty disgusting to use...).